src/services/publishing/publishing.ts

```diff
@@ -22,6 +23,48 @@ export const assertAddressIsSmartContract = async (
   if (code === "0x") throw new Error("Address is not a smart contract");
 };
 
+export const assertValidDocument = async (job: PublishingJob, network: Network): Promise<void> => {
+  if (network === "local") return;
+
+  const verify = verificationBuilder(openAttestationVerifiers, { network: network });
+  const errorMessages: Array<string> = [];
+  for (const document of job.documents) {
+    const verificationFragments = await verify(document.wrappedDocument);
+
+    // Validate for invalid document store, token registry and contract
+    const documentStatusFragments = utils.getDocumentStatusFragments(verificationFragments);
```

Comment:
can use https://github.com/TradeTrust/tradetrust-utils/blob/master/src/fragments/fragments.ts#L17

Reply:
That function checked if the document has been issued and this will 100% trigger an error as we are validating the document before issuing